utils/docker-compose/Makefile

```diff
@@ -1,49 +1,47 @@
 include $(TOPDIR)/rules.mk
 
-PKG_NAME:=docker-compose
-PKG_VERSION:=1.29.2
+PKG_NAME:=compose
+PKG_VERSION:=2.1.0
 PKG_RELEASE:=$(AUTORELEASE)
+PKG_LICENSE:=Apache-2.0
+PKG_LICENSE_FILES:=LICENSE
 
-PYPI_NAME:=docker-compose
-PKG_HASH:=4c8cd9d21d237412793d18bd33110049ee9af8dab3fe2c213bbd0733959b09b7
+PKG_SOURCE:=v$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/docker/compose/tar.gz/v${PKG_VERSION}?
+PKG_HASH:=48b20b1b75de0e517deb0e29122ecdb01982f136e68429bdf6a97121701d4274
 
 PKG_MAINTAINER:=Javier Marcet <javier@marcet.info>
-PKG_LICENSE:=Apache-2.0
-PKG_LICENSE_FILES:=LICENSE
```

Comment:
Others may prefer `PKG_LICENSE`/`PKG_LICENSE_FILES` stay here, I don’t have a strong opinion on this.

Reply:
It's true I moved those two lines up while I did the move to a Go package, mimicking [runc](https://github.com/openwrt/packages/blob/master/utils/runc/Makefile).

While I do not see any reference to this matter in the [package-policies](https://openwrt.org/docs/guide-developer/package-policies), checking the [packages](https://github.com/openwrt/packages) tree reveals most of the packages have the three `PKG_` variables together.

I kind of liked them separate since it looks like the license is related somehow to the maintainer, while it is an upstream thing entirely, just like the version or the homepage.

So, for me I'd leave it as it is, if anyone else cares and really prefers to keep it like the majority of packages, please tell me and I'll change it.